http_service/models.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import lzma
+import os
+import shutil
+from urllib.request import urlretrieve
+
+import requests
+
+from bugbug.models.component import ComponentModel
+from bugbug.models.defect_enhancement_task import DefectEnhancementTaskModel
+from bugbug.models.regression import RegressionModel
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger()
+
+MODELS = {
```

Comment:
I wonder if we couldn't reuse at least part of the central place where we define models

Reply:
Yes now that we have merged the centralization PR, we should, let me update this PR